tasks/testwasher.py

```diff
@@ -148,3 +153,61 @@ def get_tests_family(self, test_name_list):
                 test_name = test_name.rsplit('/', 1)[0]
                 test_family.add(test_name)
         return test_family
+
+
+@task()
+def generate_flake_finder_pipeline(_, n=3):
+    """
+    Verify that the jobs defined within job_files contain a change path rule.
+    """
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
    """
    Generate a child pipeline where jobs marked with `FLAKES_FINDER` run n times
    """
```